apps/dashboard/src/components/primitives/sonner-helpers.tsx

```diff
@@ -33,7 +33,7 @@ export const showSuccessToast = (message: string, position: 'bottom-center' | 't
 export const showErrorToast = (
   message: string,
   title?: string,
-  position: 'bottom-center' | 'top-center' = 'bottom-center'
+  position: 'bottom-center' | 'top-center' | 'bottom-right' = 'bottom-right'
```

Comment:
Handles: https://linear.app/novu/issue/NV-5091/toast-miss-alignment-in-the-dashboard